docs/guides/qunasys-quri-chemistry.ipynb

```diff
@@ -357,14 +360,10 @@
     ")\n",
     "\n",
     "while True:\n",
-    "    try:\n",
-    "        print(datetime.datetime.now(), job.status())\n",
-    "        time.sleep(10)\n",
-    "        if job.status() in [\"ERROR\", \"DONE\"]:\n",
-    "            break\n",
-    "    except:\n",
-    "        print(\"Retrieve job status failed.\")\n",
-    "        pass\n",
```

Comment:
I removed the exception as we actually do want the notebook to halt on error here, otherwise `job.result()` will fail immediately after. Same for L559.